src/Catalog/NuGet.Services.Metadata.Catalog.csproj

```diff
@@ -252,12 +259,18 @@
     <PackageReference Include="dotNetRDF">
       <Version>1.0.8.3533</Version>
     </PackageReference>
-    <PackageReference Include="NuGet.Packaging">
-      <Version>5.0.0-preview1.5665</Version>
+    <PackageReference Include="NuGet.Services.Entities">
+      <Version>2.41.0</Version>
+    </PackageReference>
+    <PackageReference Include="NuGet.Services.Sql">
+      <Version>2.46.0</Version>
```

Comment:
Is this version consistent with the rest of the sln?

Reply:
Yep!